docs/source/nn.rst

```diff
@@ -4,6 +4,15 @@
 torch.nn
 ===================================
 
+.. note:: The contents of this page are no longer maintained. Click here to see the `stable version of torch.nn <https://pytorch.org/docs/stable/nn/index.html>`_.
+
+
+.. role:: hidden
+    :class: hidden-section
+
+torch.nn
```

Comment:
Another reason to delete the rest of this section is to avoid link confusion. For example, our F.conv2d documentation links to `nn.Conv2d` -- we want to be sure that it is linking to the `nn.Conv2d` in `nn_convolution_layers.rst`, not here.